.github/workflows/tests-coverage.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Tests CI
+
+on:
+  pull_request:
+    branches: ['main']
```

Comment:
This will run CI test for PR raised against the main branch only.
Will it run for develop branch also?

Reply:
nope, it will be for only main, since @harshith-venkatesh  had requested it.

Reply:
I think it should run for develop also

Reply:
@harshith-venkatesh are we going to run Actions for PR that are raised against the main branch only? 
Any reason for it?

Reply:
Sorry Vivek, we need to add develop as well.
Currently we have not created branches for dev , staging, prod.
Once done we will be adding those as well

Reply:
cool , no issues